packages/website/docs/sources.md

```diff
@@ -271,13 +428,15 @@ const source = {
 
 Called to get the value of the item. The value is used to fill the search box.
 
-If you do not wish to update the input value when an item is selected, you can return `state.query`.
+If you don't want to update the input value when an item is selected, you can return `state.query`.
```

Comment:
Isn't it easier not to implement the function?

Reply:
Yes, especially that we document the default value.

Reply:
Dropped in 62cc97b.